address/encoding.go

```diff
@@ -17,10 +17,21 @@ func compressedPubKeyEncoder(w io.Writer, val any, buf *[8]byte) error {
 	return tlv.NewTypeForEncodingErr(val, "*btcec.PublicKey")
 }
 
-func compressedPubKeyDecoder(r io.Reader, val any, buf *[8]byte, l uint64) error {
+func compressedPubKeyDecoder(
```

Comment:
I'm not sure if spending time on this is really worth doing right now...
My suggestion would be to instead commit the `.vscode/settings.json` file to the repo so a ruler is automatically shown (see https://github.com/lightningnetwork/lnd/tree/master/.vscode for example).

Also, please take a look at our formatting rules, we want a much more compact formatting of the function headers: https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md